modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/constants/DeepLConstants.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.translator.deepl.internal.constants;
+
+/**
+ * @author Yasuyuki Takeo
+ */
+public class DeepLConstants {
+
+	public static final String AUTH_KEY = "auth_key";
+
+	public static final int CALL_TIMEOUT = 30;
+
+	public static final int CONNECTION_TIMEOUT = 15;
+
+	public static final int KEEP_ALIVE_DURATION = 120;
+
+	public static final int MAX_IDLE_CONNECTIONS = 200;
+
+	public static final int READ_TIMEOUT = 15;
+
+	public static final String SOURCE_LANG = "source_lang";
+
+	public static final String SUPPORTED_LANGUAGE_INQ_URL =
+		"https://api-free.deepl.com/v2/languages";
+
+	/**
+	 * Verifying the supported languages
+	 */
```

Comment:
Can you remove this comment? In general, comments that are not strictly necessary should be avoided.